WcaOnRails/config/locales/en.yml

```diff
@@ -638,6 +639,7 @@ en:
         comments: ""
         guests: ""
         status: ""
+        administrative_notes: ""
```

Comment:
It might be worth actually adding a hint here, to point out the difference between "Comments" and "Administrative notes".